app/models/url.rb

```diff
@@ -5,7 +5,6 @@ class Url < ActiveRecord::Base
   before_save :generate_short_url
 
   def generate_short_url
-    url.short = ('a'..'z').to_a.shuffle[0,5].join
-    true
+    self.short = ('a'..'z').to_a.shuffle[0,5].join
```

Comment:
Space missing after comma.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.